qiime/split_libraries_lea_seq.py

```diff
@@ -252,7 +253,7 @@ def get_cluster_ratio(fasta_seqs, min_difference_in_clusters):
 
     sorted_counts_in_clusters = sorted(
         count_lookup.iteritems(),
-        key=lambda x: x[1])
+        key=lambda x: x[1], reverse=True)
```

Comment:
Sorry, nevermind -- looks like one of the unit tests below had its expected output updated.